assets/javascripts/tests.js

```diff
@@ -264,7 +264,11 @@ function renderTestLists() {
       data: ajaxQueryParams,
       dataSrc: function (json) {
         // update heading when JSON is available
-        $('#running_jobs_heading').text(json.data.length + ' jobs are running');
+        let text = json.data.length + ' jobs are running';
+        if (json.max_running_jobs >= 0) {
```

Comment:
The value is only passed to the client side if it's not undef, so that's ok.

Reply:
I know, hence `json.max_running_jobs` might be `undefined` on the client-side. Supposedly the `>=` operator deals with it but it would be good to make it more clear that this object property might be `undefined`.

Reply:
done